conf/sahmri.config

```diff
@@ -0,0 +1,29 @@
+params {
+  config_profile_description = 'South Australian Health and Medical Research Institute (SAHMRI) HPC cluster profile.'
+  config_profile_contact     = 'Nathan Watson-Haigh (nathan.watson-haigh@sahmri.com)'
+  config_profile_url         = "https://sahmri.org.au"
+  max_memory      = 375.GB
+  max_cpus        = 32
+  max_time        = 14.d
+  igenomes_base   = '/cancer/storage/shared/igenomes/'
+}
+process {
+  executor        = 'slurm'
+  queue           = 'sahmri_prod_hpc,sahmri_cancer_hpc'
+  maxRetries      = 2
+
+  cpus            = { check_max( 2 * task.attempt, 'cpus') }
+  memory          = { check_max( 1.GB * task.attempt, 'memory') }
+  time            = { check_max( 10.m * task.attempt, 'time') }
+}
+executor {
+  queueSize       = 50
+  submitRateLimit = '10 sec'
+}
+singularity {
+  enabled         = true
+  autoMounts      = true
+  beforeScript    = 'export PATH=/apps/opt/singularity/latest/bin:${PATH}'
+  cacheDir        = '/cancer/storage/shared/simg'
+}
+cleanup = true
```

Comment:
If I set this to run by default within the config, I normally like to have a `debug` profile or something to allow a user to turn this off (e.g. to allow checking of `work/` dirs while developing a pipeline)